### DISCUSSION TITLE
听说这个网站挺有用的，求个搭子

### DISCUSSION THREAD

**Discussion by AdangCyilly** [share121/inter-knot#4587] [委托]
找一个搭子，能开麦，能长时间游玩的，好像叫固玩搭子，pls加我  245503524
![Uploading 82411ca57029540881846494aad38bbd34370638_raw.gif…]()

**Comment by AdangCyilly** [comment]
!配个图[73d90ed1792942b58a20e015d9126ecafa15ac86_raw](https://github.com/user-attachments/assets/169a91da-e2ff-4fbb-9cbe-8c78cea6add5)

**Comment by ThinkofRain1213** [comment]
玩点什么，有我也玩的可以一起

**Reply by AdangCyilly** [reply]
加我QQpls 2352742955
